src/Paket.Core/Dependencies/NuGet.fs

```diff
@@ -806,21 +806,36 @@ let private downloadAndExtractPackage(alternativeProjectRoot, root, isLocalOverr
                 | exn -> raise (Exception(sprintf "Could not download %O %O from %s." packageName version !downloadUrl, exn)) }
 
     async {
+        configResolved.Path |> Option.iter SymlinkUtils.delete
+
         do! download true 0
-        if not isLocalOverride then
+
+        match isLocalOverride, configResolved with
+        | true, ResolvedPackagesFolder.NoPackagesFolder -> return failwithf "paket.local in combination with storage:none is not supported"
```

Comment:
Maybe we add to that message "use storage: symlink instead" ?

Reply:
done